myuw/dao/term.py

```diff
@@ -391,6 +403,10 @@ def add_term_data_to_context(request, context):
 
     context['first_day'] = cur_term.first_day_quarter
     context['last_day'] = cur_term.last_day_instruction
+    context["first_day_quarter"] = cur_term.first_day_quarter
+    context["last_day_instruction"] = cur_term.last_day_instruction
```

Comment:
i suspect that instead of last_day_instruction you may want to use the end of final week
A-term: 6/24 - 7/24
B-term: 7/25 - 8/23
Autumn 2013: 9/25 - 12/13
<img width="253" alt="a-term" src="https://user-images.githubusercontent.com/9834194/26956139-7cbd43be-4c70-11e7-8392-728f17d6eaea.PNG">
<img width="255" alt="b-term" src="https://user-images.githubusercontent.com/9834194/26956143-81aa506a-4c70-11e7-8199-bd31531b09ed.PNG">
<img width="350" alt="autumn" src="https://user-images.githubusercontent.com/9834194/26956151-88408660-4c70-11e7-8054-9bc2d94feead.PNG">